composeApp/src/commonMain/kotlin/com/overdrive/cruiser/views/UserView.kt

```diff
@@ -44,7 +47,7 @@ fun UserView(userViewModel: UserViewModel, onLogOut: () -> Unit) {
     Column(modifier = Modifier.fillMaxSize().background(color = Color(0xFFF5F5F5))
     ) {
         Column(modifier = Modifier
-            .fillMaxWidth()
+            .fillMaxSize()
             .background(color = Color(0xFFF5F5F5))
             .padding(16.dp)
         ) {
```

Comment:
also would want to reduce the spacer below this line to 16.dp

Reply:
Changed spacing around profile icon.